File: app/src/main/java/com/example/shaochengyang/zillow/data/model/PropertyItem.java
package com.example.shaochengyang.zillow.data.model;


import com.google.gson.annotations.SerializedName;


public class PropertyItem{

	@SerializedName("propertycountry")
	private String propertycountry;

	@SerializedName("propertypurchaseprice")
	private String propertypurchaseprice;

	@SerializedName("propertystatus")
	private String propertystatus;

	@SerializedName("propertymortageinfo")
	private String propertymortageinfo;

	@SerializedName("id")
	private String id;

	@SerializedName("propertystate")
	private String propertystate;

	@SerializedName("propertyaddress")
	private String propertyaddress;

	@SerializedName("propertycity")
	private String propertycity;

	public void setPropertycountry(String propertycountry){
		this.propertycountry = propertycountry;
	}

	public String getPropertycountry(){
		return propertycountry;
	}

	public void setPropertypurchaseprice(String propertypurchaseprice){
		this.propertypurchaseprice = propertypurchaseprice;
	}

	public String getPropertypurchaseprice(){
		return propertypurchaseprice;
	}

	public void setPropertystatus(String propertystatus){
		this.propertystatus = propertystatus;
	}

	public String getPropertystatus(){
		return propertystatus;
	}

	public void setPropertymortageinfo(String propertymortageinfo){
		this.propertymortageinfo = propertymortageinfo;
	}

	public String getPropertymortageinfo(){
		return propertymortageinfo;
	}

	public void setId(String id){
		this.id = id;
	}

	public String getId(){
		return id;
	}

	public void setPropertystate(String propertystate){
		this.propertystate = propertystate;
	}

	public String getPropertystate(){
		return propertystate;
	}

	public void setPropertyaddress(String propertyaddress){
		this.propertyaddress = propertyaddress;
	}

	public String getPropertyaddress(){
		return propertyaddress;
	}

	public void setPropertycity(String propertycity){
		this.propertycity = propertycity;
	}

	public String getPropertycity(){
		return propertycity;
	}

	@Override
 	public String toString(){
		return 
			"PropertyItem{" + 
			"propertycountry = '" + propertycountry + '\'' + 
			",propertypurchaseprice = '" + propertypurchaseprice + '\'' + 
			",propertystatus = '" + propertystatus + '\'' + 
			",propertymortageinfo = '" + propertymortageinfo + '\'' + 
			",id = '" + id + '\'' + 
			",propertystate = '" + propertystate + '\'' + 
			",propertyaddress = '" + propertyaddress + '\'' + 
			",propertycity = '" + propertycity + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/example/shaochengyang/zillow/data/model/PropertyItem.kt
package com.example.shaochengyang.zillow.data.model

import com.google.gson.annotations.SerializedName


data class PropertyItem(

	@field:SerializedName("propertycountry")
	val propertycountry: String? = null,

	@field:SerializedName("propertypurchaseprice")
	val propertypurchaseprice: String? = null,

	@field:SerializedName("propertyuserid")
	val propertyuserid: String? = null,

	@field:SerializedName("propertystatus")
	val propertystatus: String? = null,

	@field:SerializedName("propertyusertype")
	val propertyusertype: String? = null,

	@field:SerializedName("propertymortageinfo")
	val propertymortageinfo: String? = null,

	@field:SerializedName("id")
	val id: String? = null,

	@field:SerializedName("propertystate")
	val propertystate: String? = null,

	@field:SerializedName("propertylatitude")
	val propertylatitude: String? = null,

	@field:SerializedName("propertyaddress")
	val propertyaddress: String? = null,

	@field:SerializedName("propertycity")
	val propertycity: String? = null,

	@field:SerializedName("propertylongitude")
	val propertylongitude: String? = null
)